sdk/communication/Azure.Communication.MediaComposition/src/Generated/Models/CommunicationIdentifierModelKind.cs

```diff
@@ -8,10 +8,10 @@
 using System;
 using System.ComponentModel;
 
-namespace Azure.Communication.MediaComposition
+namespace Azure.Communication
 {
     /// <summary> The identifier kind, for example &apos;communicationUser&apos; or &apos;phoneNumber&apos;. </summary>
-    public readonly partial struct CommunicationIdentifierModelKind : IEquatable<CommunicationIdentifierModelKind>
```

Comment:
we can't do this, it'd be a breaking change. let's keep it public
- https://apiview.dev/Assemblies/Review/5b1c0e6abcad4356839133822c404c91/c7381770117e4541937ad9c500e339d0?diffRevisionId=73c97198062443d5b8a60114d36d4ac1&doc=False&diffOnly=True

Reply:
I've rolled back this change, thanks!